cirq/aqt/aqt_device.py

```diff
@@ -221,13 +199,39 @@ def get_aqt_device(num_qubits: int) -> Tuple[IonDevice, List[LineQubit]]:
     return ion_device, qubit_list
 
 
-def get_default_noise_dict() -> Dict[str, Any]:
+def get_aqt_default_noise_dict() -> Dict[str, Any]:
```

Comment:
Orthogonal (no action necessary): How stable is this api?  I think one should avoid `get` in method names as they do not add much.  `aqt_default_noise_dict()` should be just as good.